src/bot/database.ts

```diff
@@ -0,0 +1,275 @@
+import {
+  PrismaClient,
+  Pole as Pole_t,
+  Thematic as Thematic_t,
+  Project as Project_t,
+} from '../../prisma';
+
+const prisma = new PrismaClient();
+
+const createPole = async (name: Pole_t['name'], emoji: Pole_t['emoji'], rolesChannelId: Pole_t['rolesChannelId']) => {
+  const pole = await prisma.pole.create({
+    data: {
+      name,
+      emoji,
+      rolesChannelId,
+    },
+  });
+
+  return pole;
+};
+
+const getPole = async (id: Pole_t['id']) => {
+  const pole = await prisma.pole.findUnique({
+    where: {
+      id,
+    },
+  });
+
+  return pole;
+};
+
+const getPoleByName = async (name: Pole_t['name']) => {
+  const pole = await prisma.pole.findUnique({
+    where: {
+      name,
+    },
+  });
+
+  return pole;
+};
+
+const getPoles = async () => prisma.pole.findMany();
+
+const deletePole = async (name: Pole_t['name']) => {
+  // Get pole
+  const pole = await prisma.pole.findUnique({
+    where: {
+      name,
+    },
+  });
+
+  if (!pole) return 'Unable to find the pole.';
```

Comment:
Shouldn't it throw an exception, that may be catch at an upper level (where we will log the error) ?